packages/web-app-files/src/App.vue

```diff
@@ -68,6 +74,15 @@ export default defineComponent({
         to: this.$refs.filesSidebar?.$el,
         revert: event === 'beforeDestroy'
       })
+    },
+    /* on click instead of on focus since child element got the higher tabindex and focuses first */
+    unselectOnClick(e) {
```

Comment:
What do you think about only interpreting clicks **below** the file list for clearing the selection model? In the area above the file list I'd prefer to treat a click on whitespace as a `whoops I missed that button that I wanted to click`. In that case I wouldn't want my current selection to be gone.